cirq/vis/heatmap.py

```diff
@@ -28,24 +27,28 @@
 
 from cirq.devices import grid_qubit
 
-QubitCoordinate = Union[Tuple[int, int], grid_qubit.GridQubit]
+QubitCoordinate = Union[Tuple[float, float], grid_qubit.GridQubit]
 
-# The value map is qubit coordinate -> a type that supports float conversion.
-ValueMap = Union[Dict[grid_qubit.GridQubit, SupportsFloat], Dict[Tuple[int, int], SupportsFloat]]
+QubitPair = Union[
```

Comment:
can this just be `Tuple[QubitCoordinate, QubitCoordinate]`?

Reply:
I guess keeping the types consistent is expressed here, similarly to `ValueMap` where we keep the key types consistent.